app/containers/Applications/ApplicationWizardConfirmation.tsx

```diff
@@ -235,6 +302,100 @@ export const ApplicationWizardConfirmationComponent: React.FunctionComponent<Pro
         Please review the information you have provided before continuing.
       </h5>
       <br />
+      {overrideActive ? (
+        <>
+          <Alert variant="danger">
+            <h4>Override Active</h4>
```

Comment:
Same comments as in the other file regarding using an `h4`. Also I'm not sure that this title would be clear enough. Do we even need a title in this page?

Reply:
I've removed the h4 and added a /<strong/>, and changed the title to 'Error Validation Override Active'